### PR TITLE
Update font-awesome to 4.5.0

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by pfanguin** [comment]
Test link : https://mf-geoadmin3.dev.bgdi.ch/update_fontawesome/?lang=fr

File: src/components/swipe/style/swipe.less [hunk 1]
```diff
@@ -1,5 +1,9 @@
 [ga-swipe] {
 
+  .ga-swipe-arrows i:first-child {
+    transform: rotate(180deg);
```
Comment (oterral) [inline src/components/swipe/style/swipe.less]: `.rotate(180deg)`

File: src/components/swipe/style/swipe.less [hunk 2]
```diff
@@ -33,7 +37,8 @@
     i {
       float: left;
       line-height: 22px;
-      margin: 0px 2px;
+      margin: 0px 0px;
+      padding-top:2px;
```
Comment (oterral) [inline src/components/swipe/style/swipe.less]: space after `:`

**Comment by oterral** [comment]
@pfanguin can you fix the 3 minor things I mentioned .


otherwise looks very good. Thx for this.

**Comment by pfanguin** [comment]
It's done

**Comment by oterral** [comment]
Can you squash please ?

**Comment by oterral** [comment]
and can you make the PR on #3038

**Pull request closed without merging**